### PR TITLE
Fix UDP and network stuff

### PR DESCRIPTION
**Pull request by baude**
Add support for better error messages and also things to help integration test pass UDP and IPv6.

### CODE REVIEW

**Review by Luap99 — COMMENTED** [review]
LGTM and fixes our weird ipv6 issue

**Review by flouthoc — APPROVED** [review]

File: src/network/core_utils.rs [hunk 1]
```diff
@@ -832,7 +832,7 @@ impl CoreUtils {
         if let Err(err) = handle
             .link()
             .add()
-            .veth(host_veth.to_string(), container_veth.to_string())
+            .veth(container_veth.to_string(), host_veth.to_string())
```
Comment (flouthoc) [inline src/network/core_utils.rs:835]: @baude Sorry i think i merged it. Just one doubt I understand that swapping `veth` order is fixing the issue but not sure how ? Is it a `kernel` bug ?

Comment (Luap99) [reply src/network/core_utils.rs:835]: I am not sure tbh. I just tried until it works. I have a script with only `ip` commands to reproduce the bridge setup. However with `ip link add` the order did not matter so it is not only the kernel. I think the `ip link` command adds some netlink attributes and our rust code does not.

**Pull request merged**